pkg/registry/core/event/storage/storage.go

```diff
@@ -73,3 +79,24 @@ var _ rest.ShortNamesProvider = &REST{}
 func (r *REST) ShortNames() []string {
 	return []string{"ev"}
 }
+
```

Comment:
> Due to etcd key schema of event (/registry/events/NAMESPACE/POD_NAME.EVENT_ID), it seems reasonable fetch events related only to involved object by prefix: /registry/events/NAMESPACE/POD_NAME, that significantly decreaces amount of keys fetched from etcd.

Is that really the case? Obviously not every event is associated with a pod.
I didn't check that now, but I'm quite convinced we have the same patter for etcd keys for events as for anything else (which is {prefix}/events/namespace/<event name> (which IIRC is some kind of hash).

If I'm not missing anything, then this PR doesn't work...